### ISSUE TITLE
99.99% rounding issue in Edge

### ISSUE THREAD

**Issue by dotnetCarpenter** [peterramsing/lost#295]
**Is this a feature request or a bug report?**
Bug report

**What is the current behavior?**

![lost_edge](https://cloud.githubusercontent.com/assets/43763/15571762/0fd50b22-233d-11e6-9c9b-cc2a1137bf81.png)
_Edge_

![lost_ie11](https://cloud.githubusercontent.com/assets/43763/15571757/05212562-233d-11e6-8398-1a3002b27376.png)
_IE11_

**If it's a bug please provide the steps to reproduce it and maybe some code samples.**

``` html
<section class="f-row f-row_12">
    <div class="f-row__cell-8"><p>This is a long cell spanning 8 cells</p></div>
    <div class="f-row__cell-4"><p>This is a short cell spanning 4 cells</p></div>
</section>
```

``` css
@lost gutter 3.4rem;
html {
  box-sizing: border-box;
  /* Change 1 rem to mean 10px in a standard browser */
  font-size: 62.5%;

  lost-center: 192rem;
}
*, *:before, *:after {
  box-sizing: inherit;
}

.f-row {
    lost-flex-container: row;
    width: 100%;
}

.f-row::after {
    lost-utility: clearfix;
}

.f-row__cell {
    lost-column: 1 flex;
}

.f-row_12 .f-row__cell-4 {
    lost-column: 4/12 flex;
    background-color: hsla(1, 73.7%, 38.8%, .5);
}
.f-row_12 .f-row__cell-8 {
    lost-column: 8/12 flex;
    background-color: hsla(0, 0%, 20%, .5);
}
```

Open IE or Edge with the above HTLM/CSS and resize the browser window a bit. You should see the second _cell_ (f-row__cell-4) jump beneath the first cell.

**What is the behavior that you expect?**
That both _cells_ are on the same row/line.

![lost_opera](https://cloud.githubusercontent.com/assets/43763/15571841/81d92e10-233d-11e6-8161-b4f36ade8d6a.png)
_Opera_

**What's the motivation or use-case behind changing the behavior?**
Fixing our grid which is based on `lost`. https://github.com/Firefund/styleguide/issues/109

**What version of LostGrid, browser and browser version are affected by this issue? Did this happen in previous versions?**
Lost: 6.8.0
Edge: 25
IE: 11

**Anything else?**
I will work on a PR which set the width of `lost-column` to `99.9%` instead of `99.99%`.

**Comment by peterramsing** [comment]
Closing for #296

**Comment by peterramsing** [comment]
Closed by #308
